modules/apps/object/object-service/src/main/java/com/liferay/object/internal/upgrade/v10_2_0/ObjectFieldDBTypeUpgradeProcess.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.object.internal.upgrade.v10_2_0;
+
+import com.liferay.portal.kernel.upgrade.UpgradeProcess;
+
+import java.sql.PreparedStatement;
+
+/**
+ * @author Jhosseph Gonzalez
+ */
+public class ObjectFieldDBTypeUpgradeProcess extends UpgradeProcess {
+
+	@Override
+	protected void doUpgrade() throws Exception {
+		try (PreparedStatement preparedStatement = connection.prepareStatement(
```

Comment:
I think you can simplify this by just using the runSQL function like we have [here](https://github.com/liferay/liferay-portal/blob/6ce0b9eb049e99a50285a324a3bd4afd76d340ab/modules/apps/object/object-service/src/main/java/com/liferay/object/internal/upgrade/v5_1_0/ObjectDefinitionUpgradeProcess.java#L21). Or better yet, avoid creating this new class and use the same function directly in the `ObjectServiceUpgradeStepRegistrator` class like we have [here](https://github.com/liferay/liferay-portal/blob/9d6035ac2ce27937aece357e1e8d91c07bd1360c/modules/apps/object/object-service/src/main/java/com/liferay/object/internal/upgrade/registry/ObjectServiceUpgradeStepRegistrator.java#L456-L463).